Client/Application/AppDelegate.swift

```diff
@@ -158,11 +158,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         
         adjustIntegration = AdjustIntegration(profile: profile)
 
-        // We need to check if the app is a clean install to use for
-        // preventing the What's New URL from appearing.
-        if getProfile(application).prefs.intForKey(IntroViewControllerSeenProfileKey) == nil {
-            getProfile(application).prefs.setString(AppInfo.appVersion, forKey: LatestAppVersionProfileKey)
```

Comment:
I think this is not exactly related to Leanplum. Are you sure this should be removed?